git/pre-commit

```diff
@@ -2,6 +2,16 @@
 
 if [ "$NOLINT" ]; then exit 0; fi
 
+if [ ! $TRAVIS = true ]
```

Comment:
> Checking if $TRAVIS is different than true
https://github.com/Vauxoo/maintainer-quality-tools/pull/166#discussion_r93695531

What are you checking ? 
if you are checking TRAVIS is different from true you must use 
`[$TRAVIS != true]`
If you use:
`[ ! $TRAVIS = true]` you're negating the result only, TRAVIS could be true but you're negating that fact